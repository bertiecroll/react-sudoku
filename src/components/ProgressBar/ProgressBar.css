.ProgressBar {
  width: 90%;
  margin-bottom: 1.5em;
  display: flex;
  justify-content: space-around;
}
