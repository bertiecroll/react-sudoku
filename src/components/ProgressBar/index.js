export { ProgressBar } from './ProgressBar'
export { default } from './ProgressBar'
