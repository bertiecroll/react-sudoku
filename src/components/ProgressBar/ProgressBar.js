import React from 'react'

import './ProgressBar.css'
import FaultCounter from 'components/FaultCounter'
import Timer from 'components/Timer'

export default function ProgressBar (props) {
  return (
    <div className="ProgressBar">
      < FaultCounter />
      < Timer />
    </div>
  )
}
