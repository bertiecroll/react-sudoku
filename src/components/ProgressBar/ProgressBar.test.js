import React from 'react'
import { shallow } from 'enzyme'
import ProgressBar from 'components/ProgressBar'

describe('ProgressBar', () => {
  it('renders FaultCounter and Timer components', () => {
    const wrapper = shallow(<ProgressBar />)
    expect(wrapper).toMatchSnapshot()
  })
})
