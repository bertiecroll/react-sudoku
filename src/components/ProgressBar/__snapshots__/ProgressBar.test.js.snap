// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar renders FaultCounter and Timer components 1`] = `
<div
  className="ProgressBar"
>
  <Connect(FaultCounter) />
  <Timer />
</div>
`;
